tests/README.md

```diff
@@ -74,10 +89,34 @@ make build-images load-images TAG=$(whoami)
 
 ## Step 3 - Run the tests
 
+### 3a - Run the tests locally
+
 ```makefile
 make test TAG=$(whoami)
 ```
 
+### 3b - Run the tests on a GKE cluster from a GCP VM
+
+This step only applies if you would like to run the tests from a GCP based VM. The VM should be created in the same
```

Comment:
Do we need to create a VM before running the script or is the VM created by the script?